Communicate/Tests/CommunicateImplementingTests.swift

```diff
@@ -55,7 +55,8 @@ final class CommunicateImplementingTests: XCTestCase {
                 let loggedEvent = self.analytics.eventsLogged.first
                 XCTAssertEqual(loggedEvent?.type, "user")
                 XCTAssertEqual(loggedEvent?.action, "pushReceived")
-                let trackInfoString = self.convertToString(self.testTrackInfo)
+                let extractTrack = self.extractTrackInfo(with: self.testTrackInfo)
```

Comment:
Because `extractTrackInfo` method is exactly the same as `extractTrackInfo` in `CommunicateImplementing` to make sure that the data is extracted as expected, I would recommend you to do this:
`let extractTrack = ["pushId": "123"]`

Reply:
That's a good point! Thank you :)